apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -75,23 +93,60 @@ class AnimationPicker extends React.Component {
     } else if (this.props.uploadInProgress) {
       return <h1 style={styles.title}>{msg.animationPicker_uploading()}</h1>;
     }
+
+    // Update text depending on the context of the animation picker
+    let imageText;
```

Comment:
a few comments on this:
- i think we should move this to a function to simplify `renderVisibleBody`
- naming nit: `imageText` makes me think this is associated with an image
- the `default` statement will break if we add any more types to `PICKER_TYPE`
- the strings here should also be translated